db/migrate/20230823083547_add_data_analyst_role_enum.rb

```diff
@@ -0,0 +1,21 @@
+class AddDataAnalystRoleEnum < ActiveRecord::Migration[7.0]
+  def up
+    execute <<-SQL
+      ALTER TYPE user_role ADD VALUE IF NOT EXISTS '#{Types::DATA_ANALYST_ROLE}';
+    SQL
+  end
+
+  def down
```

Comment:
I'd rather this down migration raise an error if there are data analysts present in the database.

Reply:
Changed to throw an exception on rollback making rollback impossible.

Reply:
Thank you :)